pkg/apis/pipeline/v1beta1/swagger.json

```diff
@@ -1471,6 +1471,24 @@
         }
       }
     },
+    "v1beta1.ResolverRef": {
```

Comment:
Just for my own understanding what is this file for? Are the changes generated or manual?

Reply:
It's generated by, I think, `./hack/openapigen.sh`

I believe `swagger.json` can be used to generate API clients for other programming languages. I don't know of any examples of folks using it, though.